pypopquiz/io.py

```diff
@@ -25,6 +25,7 @@ def verify_input(input_data: Dict) -> None:
             "round": {"type": "number"},
             "theme": {"type": "string"},
             "spacers": {"type": "boolean"},
+            "use_existing": {"type": "boolean"},
```

Comment:
Can we give this a slightly more descriptive name? Something like `use_cached_video_files`? But still then it might also apply to the downloaded YouTube videos, but I think `use_existing` is a bit too general.